tests/integration/test_obs_connection.py

```
@@ -10,65 +11,70 @@
             # Test getting scene list
             logger.info("Testing scene list...")
             scenes = await obs_client.get_scene_list()
             logger.info(f"Available scenes: {scenes}")
-            
+
         else:
             logger.error("✗ OBS connection failed!")
             return False
-            
+
     except Exception as e:
```

Ruff (0.14.14)

Do not catch blind exception: `Exception`

(BLE001)